ext/opcache/zend_file_cache.c

```diff
@@ -803,6 +803,7 @@ static void zend_file_cache_serialize_class(zval                     *zv,
 		for (i = 0; i < ce->num_interfaces; i++) {
 			SERIALIZE_STR(interface_names[i].name);
 			SERIALIZE_STR(interface_names[i].lc_name);
+			// SERIALIZE_BOOL(interface_names[i].is_optional) ?
```

Comment:
No special handling is needed, this is implicitly handled. The pointers need special handling to change their offsets, but the bool is simply a primitive.